src/backend/Dockerfile.nextstrain

```diff
@@ -39,6 +39,8 @@ RUN mkdir /ncov && \
     git remote add origin https://github.com/nextstrain/ncov.git && \
     git fetch origin master && \
     git reset --hard FETCH_HEAD
+RUN mkdir -p /ncov/auspice
+RUN mkdir -p /ncov/logs
```

Comment:
Make sure these directories always exist when we do a phylo run, so we don't get failures when trying to copy debug logs to s3